### ISSUE TITLE
REQUEST: Standardization of customization

### ISSUE THREAD

**Issue by Kreyren** [RXT0112/Zernit#90]
### Describe the feature

We need some website that allows end-users to just press a button and install packages and customization i.e using https://www.pling.com/p/1136805/

### Expected behavior

Press a button and get the required package

**Comment by Uniminin** [comment]
hey stole my idea. kek

**Comment by Kreyren** [comment]
@Uniminin No, i've implemented your problem into a long-term solution on Zernit that is expected to be painless.

**Comment by Uniminin** [comment]
alright.
